packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -787,6 +787,10 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
   void _dragEnd(_DragInfo item) {
     setState(() {
       if (_insertIndex == item.index) {
+        //No changes required if last child is being inserted into the last position
+        if(_insertIndex! + 1 == _items.length){
+          return;
```

Comment:
This check should happen outside of the setState since there is not state to set.

Reply:
suggestion resolved.